### PR TITLE
CMPS-129: Identity management system

### PR DESCRIPTION
**Pull request by antonAce**

### CODE REVIEW

File: Src/Campus.Services.Interfaces/DTO/User/UserRegistrationDto.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+namespace Campus.Services.Interfaces.DTO.User
+{
+    public class UserRegistrationDto
+    {
+        public string Email { get; set; }
+        public string UserName { get; set; }
```
Comment (profileEnumerable) [inline Src/Campus.Services.Interfaces/DTO/User/UserRegistrationDto.cs:6]: Is UserName field intended for holding Nickname, If so, consider changing to Nickname

File: Src/Campus.Services.Interfaces/DTO/User/UserViewDto.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Campus.Services.Interfaces.DTO.User
+{
+    public class UserViewDto
+    {
+        public string Email { get; set; }
+        public string UserName { get; set; }
```
Comment (profileEnumerable) [inline Src/Campus.Services.Interfaces/DTO/User/UserViewDto.cs:8]: Is UserName field intended for holding Nickname ?, If so, consider changing to Nickname

File: Src/Campus.Services/Infrastructure/ProfileService.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+            return new UserClaimsDto
+            {
+                ProfileId = user.Id
+            };
+        }
+
+        public async Task DeleteUserAsync(string id, CancellationToken token)
+        {
+            await _userManager.DeleteAsync(new User {Id = id});
```
Comment (profileEnumerable) [inline Src/Campus.Services/Infrastructure/ProfileService.cs:77]: Can we add the checking that id doesn't exist

Comment (profileEnumerable) [reply Src/Campus.Services/Infrastructure/ProfileService.cs:77]: Pass CancellationToken to DeleteAsync

File: Src/Campus.Services.Interfaces/DTO/User/UserRegistrationDto.cs (hunk 1 above)
Comment (antonAce) [reply Src/Campus.Services.Interfaces/DTO/User/UserRegistrationDto.cs:6]: The `UserName` is default field in the `Core Identity`

File: Src/Campus.Services.Interfaces/DTO/User/UserViewDto.cs (hunk 2 above)
Comment (antonAce) [reply Src/Campus.Services.Interfaces/DTO/User/UserViewDto.cs:8]: The `UserName` is default field in the `Core Identity`

File: Src/Campus.Services/Infrastructure/ProfileService.cs (hunk 3 above)
Comment (antonAce) [reply Src/Campus.Services/Infrastructure/ProfileService.cs:77]: Also need to delete navigation keys for solving broken refrences

**Review by profileEnumerable — APPROVED** [review]

**Pull request merged**